### PR TITLE
add parameter to use sim time to move_group launch

### PR DESCRIPTION
**Pull request by abake48**
Adds a launch argument to get `use_sim_time` parameter so that we can use the moveit configs in sim.

Test:

1. Launch gazebo and RViz:`ros2 launch kortex2_bringup kortex_sim_control.launch.py dof:=7 use_sim_time:=true launch_rviz:=false` 
2. In a separate terminal, launch move group node with `use_sim_time` on: `ros2 launch kinova_gen3_7dof_robotiq_2f_85_moveit_config sim.launch.py`
3. In RViz, add the `motion planning` panel and use `plan and execute` using a valid goal.

### CODE REVIEW

**Comment by moriarty** [comment]
```
ros2 launch kinova_gen3_7dof_robotiq_2f_85_moveit_config move_group.launch.py robot_ip:=xxx.xxx.xxx.xxx use_sim_time:=True
[INFO] [launch]: All log files can be found below /home/alex/.ros/log/2023-06-09-16-09-22-667724-alex-t14-41116
[INFO] [launch]: Default logging verbosity is set to INFO
[ERROR] [launch]: Caught exception in launch (see debug for traceback): launch configuration 'use_fake_hardware' does not exist
```
```
ros2 launch kinova_gen3_7dof_robotiq_2f_85_moveit_config move_group.launch.py robot_ip:=xxx.xxx.xxx.xxx use_sim_time:=True use_fake_hardware:=false
```

**Review by moriarty — APPROVED** [review]
I've pulled this branch and tested the arm works in moveit.

I had an issue with the gripper but that can be a follow up PR

File: kortex_moveit_config/kinova_gen3_6dof_robotiq_2f_85_moveit_config/launch/sim.launch.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    # Initialize Arguments
+    launch_rviz = LaunchConfiguration("launch_rviz")
+    use_sim_time = LaunchConfiguration("use_sim_time")
+    sim_ignition = LaunchConfiguration("sim_ignition")
+
+    description_arguments = {
+        "robot_ip": "xxx.yyy.zzz.www",
+        "use_fake_hardware": "false",
+        "gripper": "robotiq_2f_85",
+        "dof": "7",
```
Comment (moriarty) [inline kortex_moveit_config/kinova_gen3_6dof_robotiq_2f_85_moveit_config/launch/sim.launch.py:60]: :see_no_evil: missed this

Comment (abake48) [reply kortex_moveit_config/kinova_gen3_6dof_robotiq_2f_85_moveit_config/launch/sim.launch.py:60]: 😶

Comment (moriarty) [reply kortex_moveit_config/kinova_gen3_6dof_robotiq_2f_85_moveit_config/launch/sim.launch.py:60]: #138 fixes this

**Pull request merged**